train.py

```diff
@@ -26,6 +26,17 @@
     SEN12MSSegmentationTask,
 )
 
+# Taken from https://github.com/pangeo-data/cog-best-practices
+_rasterio_best_practices = {
+    "GDAL_DISABLE_READDIR_ON_OPEN": "EMPTY_DIR",
+    "AWS_NO_SIGN_REQUEST": "YES",
+    "GDAL_MAX_RAW_BLOCK_CACHE_SIZE": "200000000",
+    "GDAL_SWATH_SIZE": "200000000",
+    "VSI_CURL_CACHE_SIZE": "200000000",
+}
+os.environ.update(_rasterio_best_practices)
```

Comment:
Should this be moved to `if __name__ == '__main__':`? I'm not sure at what point these are used, is it fine to set them after rasterio has already been imported?

Reply:
It is fine to set after rasterio has been imported (I tested this somewhat recently)